x/reports/client/cli/tx.go

```diff
@@ -0,0 +1,65 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/desmos-labs/desmos/x/posts"
+	"github.com/desmos-labs/desmos/x/reports/internal/types"
+	"github.com/spf13/cobra"
+)
+
+// GetTxCmd set the tx commands
+func GetTxCmd(_ string, cdc *codec.Codec) *cobra.Command {
+	postsTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Reports transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	postsTxCmd.AddCommand(flags.PostCommands(
+		GetCmdReportPost(cdc),
+	)...)
+
+	return postsTxCmd
+}
+
+// GetCmdReportPost is the CLI command for reporting a post
+func GetCmdReportPost(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "report [post-id] [reports-type] [reports-message]",
```

Comment:
I think that maybe changing this to `create` might be better than `report`, this would avoid the `reports report` repetition by making it 

```
desmoscli tx reports create ...
```

What do you think?